js/samples/04.ai-apps/a.teamsChefBot/src/index.ts

```diff
@@ -87,10 +87,10 @@ const model = new OpenAIModel({
     defaultModel: 'gpt-3.5-turbo',
 
     // Azure OpenAI Support
-    azureApiKey: process.env.AZURE_OPENAI_KEY!,
-    azureDefaultDeployment: 'gpt-3.5-turbo',
-    azureEndpoint: process.env.AZURE_OPENAI_ENDPOINT!,
-    azureApiVersion: '2023-03-15-preview',
+    // azureApiKey: process.env.AZURE_OPENAI_KEY!,
```

Comment:
Our general standard is to keep both options (OAI/AOAI) uncommented and the READMEs would instruct devs to comment out whichever is not being used. We may have some discrepancies across the samples atm that need to be fixed, but we welcome all PRs for these. 

Would you mind updating to match this accordingly? Thanks in advance!